docs/source/objax/nn.rst

```diff
@@ -122,12 +122,18 @@ objax.nn
         ml = objax.nn.Sequential([objax.nn.Linear(2, 3), objax.functional.relu,
                                   objax.nn.Linear(3, 4)])
         x = objax.random.normal((10, 2))
-        y = ml(x)  # Runs all the operations (Linear -> ReLU -> Linear).
+        y = ml(x)       # Runs all the operations (Linear -> ReLU -> BatchNorm0D -> Linear).
```

Comment:
I don't see `BatchNorm0D` in the definition of `ml` two lines above

Reply:
Fixed